pages/our-work.html

```diff
@@ -18,110 +18,199 @@
             />
-          </div>
-          <!-- Dropdown projects menu, hidden, toggle visibility with JS -->
-          <div class="absolute hidden mt-2 bg-white">
-            <a
-              href="#"
-              class="block p-2 border-t border-b border-black hover:bg-gray-100"
-              >Project 1</a
-            >
-            <a
-              href="#"
-              class="block p-2 border-b border-black hover:bg-gray-100"
-              >Project 2</a
-            >
-          </div>
+          </a>
         </div>
-        <button
-          class="alt-button cursor-pointer text-white transition-transform hover:scale-110 w-[148px] h-[49px] text-2xl font-normal"
-        >
-          DONATE
-        </button>
-        <div
-          class="relative flex w-5 h-5 items-center group"
-          id="language-selector"
-        >
-          <img
-            src="/images/SVGs/globe-solid.svg"
-            aria-label="Globe Icon For Language Selection"
-            alt="Language Selection"
-            class="cursor-pointer object-contain h-5 w-5"
-          />
-          <div class="absolute left-0 w-full bg-white" id="language-options">
-            <a href="#" class="block">
-              <img
-                src="/images/SVGs/norway.svg"
-                aria-label="Norwegian Flag Language Selection"
-                alt="Norwegian Flag"
-                class="inline h-6 w-8"
-              />
-            </a>
-            <a href="#" class="block">
-              <img
-                src="/images/SVGs/unitedKingdom.svg"
-                aria-label="English Flag Language Selection"
-                alt="English Flag"
-                class="inline h-6 w-8"
-              />
-            </a>
-            <a href="#" class="block">
+        <nav class="flex flex-row items-center gap-10">
+          <button
+            id="openSearchBarDrawer"
+            class="flex gap-4 items-center cursor-pointer hover:scale-105 transition-transform"
+          >
+            <!-- JS intention: On click, remove -translate-y-full from section id="search-drawer-from-header", and add translate-y-0 -->
+            <img
+              src="/images/SVGs/Search.svg"
+              class="w-5 h-5 -mr-1"
+              aria-label="Search Button"
+              alt="Search Button"
+            />
+            <span class="text-2xl font-medium">Search</span>
+          </button>
+          <a
+            href="#"
+            class="text-2xl font-medium cursor-pointer hover:scale-105 transition-transform"
+            >News</a
+          >
+          <a
+            href="#"
+            class="text-2xl font-medium cursor-pointer hover:scale-105 transition-transform"
+            >About Us</a
+          >
+          <div
+            class="relative cursor-pointer hover:scale-105 transition-transform"
+          >
+            <div class="flex items-center">
+              <span class="text-2xl font-medium">Projects</span>
```

Comment:
Hi! in the design file it says "More" instead of "Projects". Have you got some other info from the content team?

Reply:
Hi :) That's not my work, that's the work of the one who created the navbar a long time ago. (I know it's kind of confusing because my work is inbetween in the header due to the image I had to place under the navbar that was already created). Older content will differ from the design file because the design team have made changes later, no need to worry about that, we will change these things later after finishing our tasks now.